bin/utils.js

```diff
@@ -0,0 +1,21 @@
+const shortHash = hash => hash.substring(0, 7);
+
+const getContentType = (ext) => {
+  let contentType = "application/octet-stream";
+  // TODO: Add more content types
+  if  (ext === ".css") {
```

Comment:
Yeah, good catch. I should probably add them... though when I initially did some testing with the .css at the beginning of this whole process the content-type for the fonts did not appear to matter and the browser loaded them fine... not sure why. The css was the problem child requiring the content-type be explicit.

Reply:
I have gone through several of our other buckets that I know are being referenced by other projects and checked what the content type is set to for each file type. If the content type was not `application/octet-stream` and it works in production right now I've added it. The others I figure we can leave alone for now since I know these work the way they are and we can always get more granular when we need to. Easy fix if a problem arises.